test/manifest.ts

```diff
@@ -4712,6 +4745,190 @@ version = "3.0.0"
+            draft: false,
+            version: Version.parse('1.1.0'), // minor bump, match filter
+            previousVersion: Version.parse('1.0.0'),
+            conventionalCommits: [
+              {
+                type: 'fix', // type match filter
+                scope: 'api', // scope match filter
+                notes: [],
+                references: [],
+                sha: 'commit123',
+                message: 'fix(api): something',
+                bareMessage: 'something',
+                breaking: false,
+              },
+            ],
+          },
+          {
+            title: PullRequestTitle.ofTargetBranch('main', 'main'),
+            body: new PullRequestBody([]),
+            updates: [],
+            labels: [],
+            headRefName: 'release-please/branches/main/components/c',
+            draft: false,
+            version: Version.parse('1.1.0'), // minor bump, match filter
+            previousVersion: Version.parse('1.0.0'),
+            conventionalCommits: [
+              {
+                type: 'feat', // type does not match filter
+                scope: 'api', // scope match filter
+                notes: [],
+                references: [],
+                sha: 'commit123',
+                message: 'feat(api): something',
+                bareMessage: 'something',
+                breaking: false,
+              },
+            ],
+          },
+        ]);
+      const getLabelsStub = sandbox
+        .stub(github, 'getLabels')
+        .resolves(['label-a']);
+      const createLabelsStub = sandbox.stub(github, 'createLabels').resolves();
+
+      const pullRequestNumbers = await manifest.createPullRequests();
+
+      expect(pullRequestNumbers).lengthOf(3);
+      sinon.assert.calledOnce(getLabelsStub);
+      sinon.assert.calledOnce(createLabelsStub);
+
+      sinon.assert.calledThrice(createPullRequestStub);
+      sinon.assert.calledWith(
+        createPullRequestStub,
+        sinon.match.has(
+          'headBranchName',
+          'release-please/branches/main/components/a'
+        ),
+        'main',
+        'main',
+        sinon.match.string,
+        sinon.match.array,
+        sinon.match({
+          autoMerge: undefined,
+        })
+      );
+      sinon.assert.calledWith(
+        createPullRequestStub,
+        sinon.match.has(
+          'headBranchName',
+          'release-please/branches/main/components/b'
+        ),
+        'main',
+        'main',
+        sinon.match.string,
+        sinon.match.array,
+        sinon.match({
+          autoMerge: {mergeMethod: 'rebase'},
+        })
+      );
+      sinon.assert.calledWith(
+        createPullRequestStub,
+        sinon.match.has(
+          'headBranchName',
+          'release-please/branches/main/components/c'
+        ),
+        'main',
+        'main',
+        sinon.match.string,
+        sinon.match.array,
+        sinon.match({
+          autoMerge: undefined,
+        })
+      );
```

Comment:
Asserts for the `autoMerge` parameter when `github.createPullRequest()` is called.